trail/upload/serializers.py

```diff
@@ -0,0 +1,31 @@
+from .models import Metadata
+from rest_framework import serializers
+
+
+class DynamicMetadataSerializer(serializers.ModelSerializer):
+    """
+    A ModelSerializer that takes an additional `fields` argument that
+    controls which fields should be displayed.
```

Comment:
Best docstring ever 💯 please make the rest like this, a nice clear and complete sentence! 😎